api/client/network.go

```diff
@@ -113,6 +113,7 @@ func (cli *DockerCli) CmdNetworkRm(args ...string) error {
 			status = 1
 			continue
 		}
+		fmt.Fprintf(cli.out, "%s\n", net)
```

Comment:
Should this be `fmt.Fprintln(cli.out, net)`? (I see we use `Fprintf(` in other locations, just wonder why)